packages/conf-postgresql/conf-postgresql.1/opam

```diff
@@ -4,7 +4,10 @@ authors: "Markus Mottl <markus.mottl@gmail.com>"
 homepage: "https://www.postgresql.org"
 dev-repo: "git+https://github.com/mmottl/postgresql-ocaml.git"
 license: "PD"
-build: [["pg_config"]]
+build: [
+  ["pg_config"] {os != "macos" | os-distribution != "homebrew"}
+  ["$(brew --prefix postgresql@15)/bin/pg_config"] {os = "macos" & os-distribution = "homebrew"}
```

Comment:
I think this does not work, you need to invoke it behind sh:
```suggestion
  ["sh" "-c" "\"$(brew --prefix postgresql@15)\"/bin/pg_config"] {os = "macos" & os-distribution = "homebrew"}
```
similarly on the other failing PRs